listenbrainz/webserver/templates/user/base.html

```diff
@@ -10,6 +10,7 @@ <h2 class="page-title">{{ user.musicbrainz_id }}</h2>
         <li {{ 'class=active' if active_section=='listens' }}><a href="{{ url_for('user.profile', user_name=user.musicbrainz_id) }}">Listens</a></li>
         <li {{ 'class=active' if active_section=='reports' }}><a href="{{ url_for('user.reports', user_name=user.musicbrainz_id) }}">Reports</a></li>
         <li {{ 'class=active' if active_section=='charts' }}><a href="{{ url_for('user.charts', user_name=user.musicbrainz_id) }}">Charts</a></li>
+        <li {{ 'class=active' if active_section=='top_artist_playlist' }}><a href="{{ url_for('user.top_artist_playlist', user_name=user.musicbrainz_id) }}">Top Artist Playlist</a></li>
```

Comment:
I don't think this needs to be on the user page right now. Let's make it a seperate URL, fix out usability and verbiage kinks over a few weeks and then see if we want to bring it front and center to a page that all users will see.